misc/AppImageBuilder.yml

```diff
@@ -26,28 +27,53 @@ AppDir:
     include:
       - libc6
       - libcrypt1
-      - libcurl3-gnutls
+      - libcurl4
       - libfreetype6
       - libgl1
+      - libhiredis0.14
       - libjpeg-turbo8
       - libjsoncpp1
       - libleveldb1d
       - libluajit-5.1-2
       - libopenal1
       - libpng16-16
-      - libsqlite3-0
+      - libpq5
```

Comment:
why are hiredis and postgres in here? they're close to useless for a client build

Reply:
I had to add that because if it appimage couldn't find the library.

Reply:
they should not be included in the build in the first place, try adjusting the cmake flags

Reply:
@cat-master21 `libhiredis0.14` is still there. It should be possible to disable with `-DENABLE_REDIS=FALSE`. It does not make sense to include that in MInetest.

Reply:
@SmallJoker This should now be fixed with the latest commit.

Reply:
I thought this comment was stale? Either way, would you mind adding the same flag to get rid of `libpq5` as well?

Reply:
My bad @SmallJoker I thought you mean't something else it should be solved with [minetest/minetest@`e36a595` (#13078)](https://github.com/minetest/minetest/pull/13078/commits/e36a59588afbf17bfcfe9c53feb729e45d53b60f).